src/css/projects.css

```
@@ -0,0 +1,45 @@
+.projects-page {
+  margin-top: 75px !important;
```

Do you need this !important?

```suggestion
  margin-top: 12px;
```

---

Also set to 12px to match the MemberInfoContainer margin.